text/0000-intro-video-script.md

```diff
@@ -0,0 +1,73 @@
+# Meta
+[meta]: #meta
+- Name: Intro video script
+- Start Date: 2021-05-18
+- Author(s): yaelharel
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+Improving documentation is a project priority for 2021.
+This RFC is an attempt to create an easy and clear starting point for new users and contributors.
+
+# Definitions
+[definitions]: #definitions
+
+N/A
+
+# Motivation
+[motivation]: #motivation
+
+It is currently relatively challenging for new users and contributors to understand what buildpacks are and what is the purpose of our project.
+It can be very useful to have a ~1 minute introductory video to provide high-level answers to these questions.
+We can post this video on our website and share it on social media.
+Hopefully this will help familiarize more people with our project.
+
+
+# What it is
+[what-it-is]: #what-it-is
+
+### The script
+
+When was the last time you struggled to wrangle yet another Dockerfile? As an application developer, you should only have to focus on what you do best - writing code, without having to worry about your container build strategy. Buildpacks can help you with just that! They are a better approach to building container images for applications.
```

Comment:
I think opening with a question is more engaging and triggers the viewers' curiosity.

Reply:
generally I agree, but I worry this question might alienate people. What if they haven't struggled with a Dockerfile? What if they like Dockerfile and don't understand why buildpacks are better? I guess I also worry that putting so much emphasis on Dockerfile upfront has risks. I also might be overthinking it.

Reply:
Interesting point of view.
@jromero, @natalieparellano, what do you think?

Reply:
I may also be overthinking this - but a very common people question ask is why buildpacks over dockerfiles (this came up in our user research and kubecon for eg.) More often than not, people don't struggle with Dockerfiles, mostly because it is really easy to write one (albient it is hard to write a **good** Dockerfile). 

Although not a good representation of our user base, if you look at the comments (we should also take this with a grain of salt since it is hackernews) here -> https://news.ycombinator.com/item?id=26041099, most newcomers to buildpacks fail to see why they would want to transition from Dockerfiles to buildpacks. 

I think people need a context like Joe's recent dockerfile article (https://jkutner.github.io/2021/04/26/write-good-dockerfile.html) to understand why dockerfiles can be a leaky abstraction and why they are bad. 

Maybe something like - 

> Have you ever wished your apps could be containerized automatically without having to copy paste random Dockerfile snippets in every project? As an application developer, you should only have to focus on what you do best - writing code; without having to worry about things like image security, creating minimal images or carefully crafting those Dockerfile instructions for proper caching. Buildpacks offer all of that and more!

Reply:
@samj1912, thanks a lot for your comment!!!
I really like your idea!

The only two concerns that I have are:
- Is it too detailed? Probably we don't catch all of the pain points that programers have with Dockerfiles.
I don't want viewers to not try buildpacks because we didn't mentioned their Dockerfile problem (while "When was the last time you struggled to wrangle yet another Dockerfile?" kind of covers "all of the problems").
- I think that our script is a bit long for a 1-minute video and by editing these few sentences, we're even make it longer. Maybe we should delete parts from the rest of the script? If so, I'm not sure what.

Reply:
> * Is it too detailed? Probably we don't catch all of the pain points that programers have with Dockerfiles.
>   I don't want viewers to not try buildpacks because we didn't mentioned their Dockerfile problem (while "When was the last time you struggled to wrangle yet another Dockerfile?" kind of covers "all of the problems").

I agree that we wont be able to capture all issues with Dockerfile if we get too much into the specifics but on the other hand I fear that a statement like "When was the last time you struggled to wrangle yet another Dockerfile?" maybe too abstract/generic. Our users have complained about the docs homepage being too high-level. I was hoping we could highlight some common issues that may appeal to both the app developer and operator personas, but I don't have a strong opinion on this given the time limitations.

Reply:
Yeah, I'm not sure what is the right approach.
This intro video is supposed to be very basic, but its purpose is also to make people want to try buildpacks, so maybe we should make it more specific. I'm not sure.
@sampeinado, would love to get your opinion.

Reply:
I'd say start with a simple statement rather a question that gets more to the core value proposition. This is kind of my intuition from sitting through a lot of startup pitches. Analogies/metaphors are good for helping people understand, but sometimes can get a little muddle. Better for discussion than a short pitch. 

For instance: 
Containerization technologies are making growing and scaling a software application from idea to a large deployment easier. Buildpacks make it easy to get from source code to container image.

Reply:
I've made a few changes thanks to your feedback. Please see https://github.com/buildpacks/rfcs/pull/161/commits/fab2b12a7f5048e25a18d35d5240198f5f7e7b73.